webapps/ch2-flight-search-fontend/src/components/MainForm.jsx

```diff
@@ -45,6 +53,7 @@ const MainForm  = ({preload:{airports, minDate, maxDate}, src, setSrc, dest, set
                 disableClearable
                 getOptionLabel={(option) => option.label}
                 options = {airports}
+                filterOptions={filterDestOptions}
```

Comment:
This is a good solution if people always type in the source airport first. What happens in the case if someone chooses the destination airport first?